src/k8s/cmd/k8s/k8s_x_cleanup.go

```diff
@@ -0,0 +1,41 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	cmdutil "github.com/canonical/k8s/cmd/util"
+	"github.com/canonical/k8s/pkg/k8sd/features"
+	"github.com/spf13/cobra"
+)
+
+func newXCleanupCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
+	var opts struct {
+		timeout time.Duration
+	}
+
+	cleanupNetworkCmd := &cobra.Command{
+		Use:   "network",
+		Short: "Cleanup left-over network resources",
+		Run: func(cmd *cobra.Command, args []string) {
+			ctx, cancel := context.WithTimeout(cmd.Context(), opts.timeout)
+			defer cancel()
+
+			if err := features.Cleanup.CleanupNetwork(ctx, env.Snap); err != nil {
+				cmd.PrintErrf("Error: failed to cleanup network: %v\n", err)
+				env.Exit(1)
+			}
+		},
+	}
+	cleanupNetworkCmd.Flags().DurationVar(&opts.timeout, "timeout", 5*time.Minute, "maximum time to wait")
```

Comment:
Other files say "the max time to wait for the command to execute". This should be changed for consistency. It should also be changed in k8s_x_wait_for.go

```suggestion
	cleanupNetworkCmd.Flags().DurationVar(&opts.timeout, "timeout", 5*time.Minute, "the max time to wait for the command to execute")
```